### DISCUSSION TITLE
Stable release and roadmap

### DISCUSSION THREAD

**Discussion by mawiegand** [nautilus/gateway#144] [General]
Hey,

I stumbled over this project while I was searching for a replacement of the GraphQL Federation Gateway by Apollo since JS is not an option for us. The project looks very promising to me but I am unsure if it is ready for productive use because there was not that much activity recently. Also the release version lets you assume that it is in early stage of development as it starts with `0`.

What is the status of the project? Can it be considered as stable? Is it missing any core functionality?
What are the plans for the future? Is there a roadmap or something like that?

Would love to hear from you and help with further development!

Thank you for publishing this project, @AlecAivazis! :slightly_smiling_face: 

Cheerio!
Marcel

**Comment by AlecAivazis** [comment]
Hey @mawiegand! 

In short, the project is still alive and actively maintained. It's being used at a companies in production should be considered good for general use. The core feature set is in place apart from subscription support. There are some minor features that I have been planning on, I just have been distracted working on other projects. If you want to help build out some more features, I would love to provide any guidance you need.

**Reply by mawiegand** [reply]
Thanks for your fast response!

Nice to hear that! We will have a discussion about the usage of this project tomorrow.
But I think thanks to your positive feedback chances are good that we are going to use it. :slightly_smiling_face:
